public/proto/frameserver.proto

```diff
@@ -8,10 +8,14 @@ service FrameServer {
 
     // Returns a list of the names and durations of all animations in the scene.
     rpc FetchSceneData (EmptyRequest) returns (FetchSceneDataResponse);
+    
+    // Returns when the Manim script changes
```

Comment:
It makes more sense to say this is called when the script changes, since it doesn't return anything.